packages/datadog_flutter_plugin/android/src/test/kotlin/com/datadoghq/flutter/DatadogSdkPluginTest.kt

```diff
@@ -30,11 +30,15 @@ import io.flutter.embedding.engine.plugins.FlutterPlugin
 import io.flutter.plugin.common.BinaryMessenger
 import io.flutter.plugin.common.MethodCall
 import io.flutter.plugin.common.MethodChannel
+import io.mockk.mockkStatic
 import io.opentracing.util.GlobalTracer
 import org.junit.jupiter.api.AfterEach
 import org.junit.jupiter.api.BeforeEach
 import org.junit.jupiter.api.Test
 import org.junit.jupiter.api.extension.ExtendWith
+import org.mockito.MockedStatic
+import org.mockito.Mockito
+import org.mockito.Mockito.mockStatic
```

Comment:
seems like those 3 are not used. And if Mockito has `mockStatic`, then why do we bring `mockk`?

Reply:
Ugh, yeah (I'm surprised ktlint and detekt didn't call this out....).

It's because Mockito's mockStatic doesn't work with Android. I don't know the particulars but when you try to use it throws an exception. mockk's version works with Android (and to be honest, I kinda like mockk better... I might switch everything over at some point).